ansible/dom0-bootstrap.yml

```diff
@@ -0,0 +1,18 @@
+---
+- include: ansible-version.yml
+
+- hosts: gh # TODO: it eventually should become `all`
+  become: false
+  remote_user: root
+  gather_facts: false
+  pre_tasks:
+    - name: bootstap python
+      raw: if [ ! -x /usr/bin/python ]; then apt-get update && apt-get -y install python-simplejson python-apt; fi
```

Comment:
That's interesting, GH Debian 8.7 and Ubuntu 16.04 images have `python` and ansible works OK, although I was pretty sure that this `pre_task` is there for a reason.
Seems, my memory is betraying me :-)